include/boost/mpi/detail/forward_iprimitive.hpp

```diff
@@ -42,7 +42,7 @@ class forward_iprimitive
     
     /// loading of arrays is forwarded to the implementation archive
     template<class T>
-    void load_array(serialization::array<T> & x, unsigned int file_version )
+    void load_array(serialization::array_wrappe<T> & x, unsigned int file_version )
```

Comment:
The code is obviously not exercised. The tests are all green if the patch is applied. So I guess test is needed then.